src/audio/kpb.c

```diff
@@ -75,6 +75,8 @@ static void kpb_clear_history_buffer(struct hb *buff);
 static void kpb_free_history_buffer(struct hb *buff);
 static bool kpb_has_enough_history_data(struct comp_data *kpb,
```

Comment:
s/veryfining/verifying in the commit message. I have added the comment here because I don't know where to write the comments for commit message. 

More than that, please add a small explanation why this commit is needed. For example: We prepare the code for supporting multiple sampling widths. 

This greatly helps reviewers.

Reply:
Sure, done.